promscale/manage-data/delete-data.md

```diff
@@ -121,7 +121,11 @@ SELECT compress_chunk(show_chunks('prom_data.container_cpu_load_average_10s', ol
 You can delete all trace data from the database using the
 `ps_trace.delete_all_traces()` function. This function restores the schema to a
 default state, truncates the tables in the `_ps_trace` schema, and deletes all
-the data.
+the data. You can only run the function when the Promscale Connector is not
+running.
+Note: To run this function, stop the Promscale Connector, connect to the
+database and run `SELECT ps_trace.delete_all_traces;`, then start the Promscale
+Connector.
```

Comment:
the data. You can run the function only when the Promscale Connector is not
running.

<highlight type="note">
To run this function, stop the Promscale Connector, connect to the database and
run `SELECT ps_trace.delete_all_traces;`, then start the Promscale Connector.
</highlight>